modin/numpy/utils.py

```diff
@@ -19,12 +19,7 @@
 _INTEROPERABLE_TYPES = (pd.DataFrame, pd.Series)
 
 
-def try_convert_from_interoperable_type(obj):
+def try_convert_from_interoperable_type(obj, copy=True):
     if isinstance(obj, _INTEROPERABLE_TYPES):
-        new_qc = obj._query_compiler.reset_index(drop=True)
-        new_qc.columns = range(len(new_qc.columns))
-        obj = np.array(
-            _query_compiler=new_qc,
-            _ndim=2 if isinstance(obj, pd.DataFrame) else 1,
-        )
+        obj = np.array(obj, copy=copy)
```

Comment:
if this is all what this function does, can we replace its calls with `numpy.asarray(obj, copy=copy)` everywhere instead?

Reply:
Not quite, since this is checking if object is an interoperable type, and making it a `modin.numpy.array` (not a `numpy.array`), and returning the original object if it isn't.